mne/commands/mne_setup_source_space.py

```diff
@@ -0,0 +1,135 @@
+#!/usr/bin/env python
+"""Set up bilateral hemisphere surface-based source space with subsampling.
+
+Examples
+--------
+.. code-block:: console
+
+    $ mne setup_source_space --subject sample
+
+
+ .. note : Only one of --ico, --oct or --spacing options can be set at the same
+           time. Default to spacing = 7mm.
```

Comment:
This is an opportunity to set better defaults. I'd rather make the default `oct6` here.